### PR TITLE
main: Add explicit sort ordering for 'LandingJobStatus' (bug 1893455)

### PR DESCRIPTION
**Pull request by ahochheiden**

### CODE REVIEW

File: src/lando/main/models/landing_job.py
Comment (zzzeid) [inline src/lando/main/models/landing_job.py:232]: nit
```suggestion
        q = q.annotate(status_order=ordering).order_by("-status_order", "-priority", "created_at")
```

**Review by zzzeid — CHANGES_REQUESTED** [review]
Whenever a model is changed, `lando makemigrations` should be run to generate a migration file. Meant to add that as a make command as well.

**Comment by ahochheiden** [comment]
> Whenever a model is changed, `lando makemigrations` should be run to generate a migration file. Meant to add that as a make command as well.

Done.

File: src/lando/main/models/landing_job.py
Comment (ahochheiden) [reply src/lando/main/models/landing_job.py]: Works for me.

**Review by zzzeid — APPROVED** [review]
lgtm!

one nit: if you look at previous commit message history, the first line is usually kept pretty short (on my machine, under 72 characters), and it matches the PR title before merge. You can keep it brief and put any additional detail in the rest of the message. On my machine, vim automatically breaks the line at that point. I think we have some exceptions here and there, and it's more of a convention/preference so I'll leave it up to you. We should probably have a proper styleguide for commits somewhere.

Looks like the average first line length is about 65 characters.

**Pull request merged**
